packages/formatter-json/README.md

```diff
@@ -0,0 +1,14 @@
+# Formatter json (`@sonarwhal/formatter-json`)
+
+The `json` does a `JSON.stringify()` of the results. Output is not very user
+friendly:
+
+![Example output for the json formatter](images/json-output.png)
```

Comment:
Move `images/` into `src/` so that they are copied into `dist/` and get publish with the rest.

Reply:
Done, I've changed the `package.json`